### ISSUE TITLE
May I submit a Chinese readme file ？

### ISSUE THREAD

**Issue by W-Qing** [transitive-bullshit/nextjs-notion-starter-kit#145]
This project is awesome. If possible, I would like to provide a Chinese document to help Chinese users use this project better.

**Comment by yanyanlongxia** [comment]
I also agree with this idea. It will be useful and bring this repo many stars.

**Comment by transitive-bullshit** [comment]
@W-Qing @yanyanlongxia that would be amazing 👍 😄

**Comment by allinu** [comment]
Chinese translation has been added and a PR has been submitted Please check @transitive-bullshit #147

**Comment by W-Qing** [comment]
> @W-Qing @yanyanlongxia that would be amazing 👍 😄

@transitive-bullshit  Hi, Travis, sorry for the late reply. I have completed the translation of the [Chinese document](https://github.com/transitive-bullshit/nextjs-notion-starter-kit/pull/149), but there are two checks from Vercel. Please deal with it. If there is no problem, I will close this issue later.😄
